docs/develop/01_sdk/04_integrate/04_extension_api.md

```diff
@@ -151,43 +139,69 @@ The following is an example of how you can use this method in a React applicatio
   </TabItem>
 </Tabs>
 
-## Well-Known DID Configuration
+## DID Configuration
+
+You need [an existing DID configuration](../../01_sdk/02_cookbook/01_dids/00_generate_keys.md) setup for your application to communicate with KILT extensions.
 
 This library helps set up the [Well-Known DID Configuration](https://identity.foundation/.well-known/resources/did-configuration/) as required by the [KILT Credential API specification](https://github.com/KILTprotocol/spec-ext-credential-api).
 
 ### Using the CLI Tool
 
-This library includes a CLI tool to create a [DID Configuration Resource](https://identity.foundation/.well-known/resources/did-configuration/#did-configuration-resource). This resource is necessary to establish a secure, end-to-end encrypted communication channel between a conforming browser extension and the application backend.
+This library includes a CLI tool to create a [DID Configuration Resource](https://identity.foundation/.well-known/resources/did-configuration/#did-configuration-resource).
+This resource is necessary to establish a secure, end-to-end encrypted communication channel between a conforming browser extension and the application backend.
 
-:::warning KILT Account
+Run the CLI tool using Yarn as follows:
 
-The `createDidConfig` CLI tool **only** works if you installed the package with Yarn.
+<Tabs groupId="yarn-node-choice">
+  <TabItem value='yarn' label='Yarn' default>
 
-:::
+  ```bash
+  yarn createDidConfig --did <your DID> \
```

Comment:
we may note that this only works after having added the library to your current project, while npx always works